### DISCUSSION TITLE
Welcome to xeus-python-kernel Discussions!

### DISCUSSION THREAD

**Discussion by martinRenou** [jupyterlite/xeus-python-kernel#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by omert** [comment]
Hi! I have a question. After loading the xeus kernel, I tried to access the FS in the console on Chrome. I was able to get the object and access it, but not the /drive subdirectory (see the attached screenshot). What am I doing wrong? PS - in Firefox it works fine.


![Screenshot from 2022-08-20 14-38-01](https://user-images.githubusercontent.com/14848/185766804-4dbc52a5-d2db-42bf-8b80-2888958bdf36.png)

**Reply by martinRenou** [reply]
Hi @omert.

The URL `/api/drive/?m=readdir` looks like the old service worker API. So either you're using a mismatched jupyterlite/jupyterlite-xeus-python couple, or you have an outdated service worker registered. 

I'd suggest:
- update jupyterlite and jupyterlite-xeus-python to the latest versions
or/and
- if you know how to unregister an old service worker, you can try to do it and refresh jupyterlite. Otherwise, you can also use another port for your server and try again.

**Reply by omert** [reply]
Thanks a lot @martinRenou! I tried to switch to the current jupyterlite version (instead of just using `jupyter lite build`), but was unable to get it to work with my development version of xeus. I also could not find documentation for how to do this. Any help would be much appreciated, and thanks a lot again!

**Reply by martinRenou** [reply]
Sorry, I did not mean making dev installs, I meant using the latest releases. For jupyterlite-xeus-python the latest dev install would be the same as the latest release anyway.

Did you want to have dev installs?

**Reply by omert** [reply]
Thanks again, @martinRenou, for being so responsive. I am currently using whatever conda is delivering to me. When I run `jupyter lite --version` I get 0.1.0b12. I understood from your previous reply that I should run the dev install, but I am happy with anything that works. What might help is to have better documentation on the xeus repo (see issue I opened there) so that there is a complete set of instructions that one can follow to get things to work. Thanks a lot!
